File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/CSharp/Silverlight/1049/BusinessApplication/BA.Web.10.0/Models/Shared/User.shared.cs
namespace $safeprojectname$
{
    /// <summary>
    /// Разделяемый класс, который расширяет тип User и добавляет общие свойства и методы.
    /// Эти свойства и методы будут доступны как в серверном, так и в клиентском приложениях.
    /// </summary>
    public partial class User
    {
        /// <summary>
        /// Возвращает отображаемое имя пользователя, которое по умолчанию равно значению свойства FriendlyName.
        /// Если свойство FriendlyName не задано, возвращается имя пользователя.
        /// </summary>
        public string DisplayName
        {
            get
            {
                if (!string.IsNullOrEmpty(this.FriendlyName))
                {
                    return this.FriendlyName;
                }
                else
                {
                    return this.Name;
                }
            }
        }
    }
}

File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/VisualBasic/Silverlight/1042/BusinessApplication/BA.Web/Models/Shared/User.shared.vb
Namespace Web

    Partial Public Class User
        ''' <summary>
        ''' 사용자 표시 이름(기본값: FriendlyName)을 반환합니다.
        ''' FriendlyName을 설정하지 않은 경우 사용자 이름이 반환됩니다.
        ''' </summary>
        Public ReadOnly Property DisplayName() As String
            Get
                If Not String.IsNullOrEmpty(Me.FriendlyName) Then
                    Return Me.FriendlyName
                Else
                    Return Me.Name
                End If
            End Get
        End Property
    End Class
End Namespace